modules/UI/videolayout/SmallVideo.js

```diff
@@ -85,7 +85,6 @@ function SmallVideo(VideoLayout) {
     this.audioStream = null;
     this.VideoLayout = VideoLayout;
     this.videoIsHovered = false;
-    this.hideDisplayName = false;
```

Comment:
This little bit looked like dead code that was still being referenced in SmallVideo#updateView. Prove me wrong, please.